shared/src/common/enums/validation/validation.ts

```diff
@@ -0,0 +1,2 @@
+export { GroupValidationRule } from '../group/group-validation-rule.enum';
```

Comment:
delete this folder at all `shared/src/common/enums/validation`
since this is an empty now